src/stories/5-custom/iframe/iframe.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import { ArgTypes, Stories, Title } from "@storybook/addon-docs";
+import { Meta, StoryObj } from "@storybook/react";
+import { IIframeSchema } from "../../../components/custom/iframe/types";
+import {
+	CommonCustomStoryProps,
+	DefaultStoryTemplate,
+	OVERRIDES_ARG_TYPE,
+	OverrideStoryTemplate,
+	ResetStoryTemplate,
+} from "../../common";
+import { ChildDefault, ChildResize, ChildValidateSubmission, ChildValidation } from "./doc-elements";
+
+const meta: Meta = {
+	title: "Custom/Iframe",
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Iframe</Title>
+					<p>Serves external content through an iframe.</p>
+					<ArgTypes of={Default} />
+					<Stories includePrimary={true} title="Example" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...CommonCustomStoryProps("iframe"),
+		label: { table: { disable: true } },
+		validationTimeout: {
+			description:
+				"Max. duration in seconds before a validation attempt is considered invalid. Use -1 to skip validation.",
+			table: {
+				type: {
+					summary: "number",
+				},
+				defaultValue: { summary: "2000" },
+			},
+		},
+		src: {
+			description: "URL of the page to serve in the component.",
+			table: {
+				type: {
+					summary: "string",
+				},
+				defaultValue: { summary: null },
+			},
+		},
+	},
+};
+export default meta;
+
+const host = `${window.location.protocol}//${window.location.host}`;
+
+export const Default = DefaultStoryTemplate<IIframeSchema>("iframe-default").bind({});
```

Comment:
the story doesn't load in the main autogenerated doc page for me, are you encountering this too? could consider hiding it completely and pointing to the stories in the sidebar instead

Reply:
seems like it could have pulled off a mini-DDOS on itself, I updated to include only the default story instead